content/faq/serverless.md

```diff
@@ -209,36 +223,53 @@ functions:
 
 > info **Hint** To learn more about the Serverless framework, visit the [official documentation](https://www.serverless.com/framework/docs/).
 
-With this place, we can now navigate to the `main.ts` file and update our bootstrap code with the required boilerplate:
+With that in place, we can now create a file called `lambda.ts` inside `src` and place the necessary boilerplate:
 
 ```typescript
 import { NestFactory } from '@nestjs/core';
 import serverlessExpress from '@vendia/serverless-express';
-import { Callback, Context, Handler } from 'aws-lambda';
+import type { Callback, Context, Handler } from 'aws-lambda';
 import { AppModule } from './app.module';
 
 let server: Handler;
 
 async function bootstrap(): Promise<Handler> {
   const app = await NestFactory.create(AppModule);
+
   await app.init();
 
   const expressApp = app.getHttpAdapter().getInstance();
+
   return serverlessExpress({ app: expressApp });
 }
 
+// This is a simple trick inspired by this article
+// https://aws.amazon.com/pt/blogs/compute/using-node-js-es-modules-and-top-level-await-in-aws-lambda/
+// This can save you up to one second from cold start.
+const delayedFactory: Promise<Handler> = Promise.resolve()
+  .then(() => bootstrap())
+  .then(app => {
+    server = app;
+
+    return app;
+  });
```

Comment:
Same here https://github.com/nestjs/docs.nestjs.com/pull/2428/files#r934201552

Reply:
Should I move this to runtime optimization or just remove it?